src/models/events/event-models.js

```diff
@@ -41,16 +44,39 @@ function remove(id) {
   return db("Events").where({ id }).del();
 }
 
-function findInvitedEvents(id) {
+function findUsersForEvent(id) {
```

Comment:
Looks like this function addresses the concern I had regarding losing functionality.